client/src/components/TreeTable/TreeTable.js

```diff
@@ -40,148 +32,122 @@ const styles = theme => ({
     backgroundColor: '#fff',
     boxShadow: '0 -2px 5px rgba(0,0,0,0.15)'
   }
-})
+});
 
 class TreeTable extends Component {
-
   constructor(props) {
-    super(props)
+    super(props);
     this.state = {
       detailsPane: false,
-    }
-    this.onChangeRowsPerPage = this.onChangeRowsPerPage.bind(this)
+      page: 0
+    };
   }
 
   componentDidMount() {
     const payload = {
-      page: this.props.page,
+      page: this.state.page,
       rowsPerPage: this.props.rowsPerPage,
       order: this.props.order,
       orderBy: this.props.orderBy
-    }
-    this.props.getTreesAsync(payload)
-  }
-
-  handleSelection = (e) => {
-
+    };
+    this.props.getTreesAsync(payload);
   }
 
-  handleClick = (event, id) => {
-    const { selected } = this.state
-    const selectedIndex = selected.indexOf(id)
-    let newSelected = []
-    if (selectedIndex === -1) {
-      newSelected = newSelected.concat(selected, id)
-    } else if (selectedIndex === 0) {
-      newSelected = newSelected.concat(selected.slice(1))
-    } else if (selectedIndex === selected.length - 1) {
-      newSelected = newSelected.concat(selected.slice(0, -1))
-    } else if (selectedIndex > 0) {
-      newSelected = newSelected.concat(
-        selected.slice(0, selectedIndex),
-        selected.slice(selectedIndex + 1),
-      )
-    }
-    this.setState({ selected: newSelected })
-  }
-
-  toggleDrawer = (event, id) => {
-    event.preventDefault();
-    if (id !== undefined) this.props.getTreeAsync(id);
+  toggleDrawer = id => {
+    console.log(id);
```

Comment:
Needed?

Reply:
I think it was leftover from the original toggleDrawer function :) but sure